resources/user-guides/attach-data.md

```diff
@@ -110,25 +110,30 @@ After you complete your configuration, verify that your rule looks like the foll
 2. Rebuild your Launch property and deploy it to the correct Environment.
 {% endtab %}
 
-{% tab title="Target" %}
-### Attaching additional Target request objects to Target to retrieve location events
+{% tab title="Target" %} <a name="Target-Attach-Data-Tab"></a>
+
+{% hint style="warning" %}
+Attach data events apply to the Target Extension only. They do not apply to the Target VEC Extension. 
+{% endhint %}
+
+### Attaching additional data to Target retrieve location events <a name="Target-Attach-Data-Retrieve-Location"> </a>
 
 The following sample shows how to _attach data_ to all outgoing `retrieveLocationContent` Target network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#target-create-rule)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#target-select-an-event)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#target-define-the-action)
-4. [Save and rebuild the property](attach-data.md#target-save-the-rule-and-rebuild-your-property)
+1. [Create a new **Rule**](#target-create-rule)
+2. [Select the **Event** you want to trigger the rule](#target-select-an-event)
+3. [Select the **Action** to attach data and define your payload](#target-define-the-action)
+4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property)
```

Comment:
Please add a period to the end of each step, like this:
"4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property)."